flyrs/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "flyrs"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "flyrs"
+crate-type = ["cdylib"]
+
+[dependencies]
+pyo3 = "0.21.2"
```

Comment:
I want to see if this works with `abi3-py38`. This will help a lot of building the packaging.

**Note**: This can be a follow up item.

Reply:
Sure, updated TODO list at https://github.com/flyteorg/flytekit/pull/2328#issue-2226044220